docs/try-it-out/longer-introduction.md

```diff
@@ -4,36 +4,36 @@ description: Quickstart covering key concepts in n8n.
 contentType: tutorial
 ---
 
-# A slightly longer introduction
+# Your first workflow
 
-This guide shows you how to automate a task using a [workflow](/glossary.md#workflow-n8n) in n8n, explaining key concepts along the way. You will:
+This guide will show you how to construct a [workflow](/glossary.md#workflow-n8n) in n8n, explaining key concepts along the way. You will:
 
-* Create a workflow from scratch. You'll build a workflow that runs every week to get data from NASA, filters the data, and generates two reports.
+* Create a workflow from scratch. 
 * Understand key concepts and skills, including:
     * Starting workflows with trigger nodes
     * Configuring [credentials](/glossary.md#credential-n8n)
-    * Manipulating data
+    * Processing data
     * Representing logic in an n8n workflow
     * Using [expressions](/glossary.md#expression-n8n)
 
+This quickstart uses [n8n Cloud](/manage-cloud/overview.md), which is recommended for new users. A free trial is available - if you haven't already done so, [sign up](https://app.n8n.cloud/register) for an account now.
 
-## Step one: Sign up for n8n
+## Step one: Create a new workflow
 
-This quickstart uses [n8n Cloud](/manage-cloud/overview.md). A free trial is available for new users. If you haven't already done so, [sign up](https://app.n8n.cloud/register) for an account now.
+When you open n8n, you'll see either:
 
-## Step two: New workflow
+* An empty workflow: if you have no workflows and you're logging in for the first time. Use this workflow.
+* The **Workflows** list on the **Overview** page. Select the <span class="inline-image">![universal create resource icon](/_images/common-icons/universal-resource-button.png){.off-glb}</span> **button** to create a new workflow.
 
---8<-- "_snippets/try-it-out/new-workflow.md"
 
-
-## Step three: Add a trigger node
+## Step two: Add a trigger node
 
 n8n provides two ways to start a workflow:
 
-* Manually, by selecting **Test Workflow**, or from the CLI if you installed n8n with npm or Docker.
+* Manually, by selecting **Test Workflow** (or from the CLI if you installed n8n with Docker).
```

Comment:
I wonder if we should even mention the CLI at this point. That seems like such a niche for this level of introduction.

Reply:
yes, i had the same thought. Will remove